docs/pages/management/operations/db-ca-rotation.mdx

```diff
@@ -0,0 +1,225 @@
+You should not rotate the `db` or `db_client` CA in parallel with other CA
+rotations.
+Rotating in parallel with other CA types, especially the `host` CA, is more
+complicated, more likely to cause problems, and does not save you any effort.
+
+However, if you want to rotate both the `db` and `db_client` CA, then you can
+(and should) save yourself some effort by rotating both of them at the same
+time.
+
+You can rotate both `db` and `db_client` certificate authorities in parallel
+by transitioning both CAs through each phase step.
+Just use the `tctl auth rotate` command for each step with `--type=db` and
+then `--type=db_client`.
+</Details>
+
+```code
+$ tctl auth rotate --manual --type=<Var name="type" description="Certificate authority to rotate"/> --phase=init 
+Updated rotation phase to "init". To check status use 'tctl status'
+```
+
+Use `tctl` to confirm that there is an active rotation in progress:
+```code
+$ tctl status
+```
+
+<Admonition type="warning" title="Avoid losing access to databases">
+If you are rotating the `db_client` CA and you do not want to lose access to
+your self-hosted databases in the next phase, you should reconfigure your 
+databases now.
+You do not need to reconfigure databases at this point if you are rotating only
+the `db` CA, although there is no harm in doing so.
+
+Consult the appropriate
+[Teleport Database Access Guide](../../database-access/guides) for your
+databases before proceeding to the `update_clients` rotation phase.
+</Admonition>
+
+## Step 2/4. Update clients
+
+Execute the transition from `init` to `update_clients`:
+
+```code
+$ tctl auth rotate --manual --type=<Var name="type" description="Certificate authority to rotate"/> --phase=update_clients 
+# Updated rotation phase to "update_clients". To check status use 'tctl status'
+```
+
+If you are rotating the `db_client` CA, this is the phase where the Teleport
+Database Service will start using client certificates issued by the new CA to
+connect to databases.
+You should verify that you can still access your databases after transitioning
+to this phase.
+
+If you lose connectivity to your databases, you can reconfigure your
+databases to restore access.
+If that does not restore access or you are unable to reconfigure a database,
+then [roll back](#rollback) to the old certificate authority.
+
+## Step 3/4. Update servers
+
+Execute the transition from `update_clients` to `update_servers`:
+
+```code
+$ tctl auth rotate --manual --type=<Var name="type" description="Certificate authority to rotate"/> --phase=update_servers 
+# Updated rotation phase to "update_servers". To check status use 'tctl status'
+```
+
+This phase does not actually do anything for `db` or `db_client` CA rotation,
+but it's required as part of the generalized Teleport CA rotation process before
+we can finish the rotation.
+
+## Step 4/4. Finish the rotation
+
+<Admonition type="warning" title="Avoid losing access to databases" CA>
```

Comment:
Since this admonition takes up most of Step 4, I think it's okay to make this regular body text.

Reply:
removed admonition wrapper: 3e8e870d5317ca9aa50c8044ff98f7a17dcbc25f